wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -119,6 +120,11 @@ getAccount accMod accId = do
             unknownMemAddrs = filter (`notElem` dbAddrs) relatedMemAddrs
         dbAddrs <> unknownMemAddrs
 
+getAccount :: MonadWalletWebMode m => AccountId -> m CAccount
+getAccount accId = do
```

Comment:
Do we really need this? I think there's only on use of it outside of this module, we could just use a single `getAccount` that takes the `(WalBalancesAndUtxo, CachedCAccModifier)`.